### DISCUSSION TITLE
How to calculate trade penalties?

### DISCUSSION THREAD

**Discussion by huey735** [bisq-network/bisq#5324] [General]
In an effort to [improve Support and Mediation](https://github.com/bisq-network/projects/issues/48), the @bisq-network/support has been working on standardizing the most we can so that the traders can get as close as possible as a single response from support agents, mediators and arbitrators.

Here I'd like to have your input regarding how to calculate trade penalties.
With each pending trade, each trader spends **mining and trading fees**, **opportunity** and **time**. So in the event the that the trade goes beyond the allocated trading period it seems fair to reimburse the party in the right.

For that purpose [this table](https://github.com/bisq-network/support/blob/master/docs/trade-penalties.md) was created. Please take a look at it and discuss bellow.

**Comment by Conza88** [comment]
Things that jump out to me:

**Buyer does not respond to mediator for 48 hours** = why? He doesn't give AF? Did not notice notification? Was away with RL? Or options trade and now quiet about it. If any intent becomes apparent/obvious that there's ill will etc. 

**Buyer is asking for selfie, copy of ID of seller, home address.** = there's been times where the other party did not setup Transferwise correctly (set one currency as default), so this was getting asked. I'd pass that on and help them fix it. I get that this is different. 

**Buyer uses BTC, Bisq, Bitcoin, BTC Tx ID, etc as a payment reference** = loose lips sink ships. 25%? I'm inclined to say, not only are you dead to me, that person should lose it all. Seriously. You put others accounts in jeopardy as likely what happened with TW.

**Buyer uses unagreed potentially concerning payment reference that such as 'payment for services', 'house rewire', 'consultancy'** = Haha, yes. Hence need to improve comm's/notifications for traders (not just agents).

**Buyer uses unagreed innocuous payment reference that such as holiday fund, flowers without agreement.** = this is 5%, seen as less worse than the above for 10%? Because looks like business orientated above vs. personal? 

**Buyer attempts to debit sellers account using their bank details**
**Buyer raises request for refund from Revolt / TransferWise due to buyer using their account to sell them Bitcoin.** = buyer losing 100% trade, but keeps security deposit. I'd also suggest losing security deposit. 

**Buyer does not make payment in trade window** = don't want to incentivise escalations. Would preface if no comms from willing to wait. e.g. TW transfer and went 4 days instead of 1, through no fault of my own. Other was ok to wait, but if this existed, might have wanted to raise to get %.

**Reply by MwithM** [reply]
> Buyer does not respond to mediator for 48 hours = why? He doesn't give AF? Did not notice notification? Was away with RL? Or options trade and now quiet about it. If any intent becomes apparent/obvious that there's ill will etc.

I think that the main reason for that is when the security deposit first confirmation is not received on the buyer's end. That bug is quite common and we can't expect than new users know that they need to look at a blockchain explorer to check if the transaction has been confirmed.

>Buyer uses BTC, Bisq, Bitcoin, BTC Tx ID, etc as a payment reference = loose lips sink ships. 25%? I'm inclined to say, not only are you dead to me, that person should lose it all. Seriously. You put others accounts in jeopardy as likely what happened with TW.

25% of the trade amount, when most of the time security deposits are 15% of the trade amount, is basically wiping the security deposit.

**Reply by pazza83** [reply]
Hi @Conza88 

My thoughts on your comments are:

> **Buyer does not respond to mediator for 48 hours** = why? He doesn't give AF? Did not notice notification? Was away with RL? Or options trade and now quiet about it. If any intent becomes apparent/obvious that there's ill will etc.

Who knows why they have not responded. In any case they should lose 25% of deposit amount. Same for buyer or seller that is unresponsive. I would not want to ascertain if there is ill will or not. Either way they lose the 25%. Only exception is if there is a bug with Bisq that they can prove at a later date with arbitrator.

> **Buyer is asking for selfie, copy of ID of seller, home address.** = there's been times where the other party did not setup Transferwise correctly (set one currency as default), so this was getting asked. I'd pass that on and help them fix it. I get that this is different.

I think if any of the above info is needed it should be provided within the payment method. Eg TransferWise US would require address of users due to US regulations imposed on banks, same if ACH or Wire Transfer US was used.  If the above info is not required then traders requesting it should be penalized. For example Bisq should avoid having traders using the platform that are registered in their home country as crypto traders and can take payment from users they have KYD'd. This behavior is prevalent on other P2P exchanges. 

> **Buyer uses BTC, Bisq, Bitcoin, BTC Tx ID, etc as a payment reference** = loose lips sink ships. 25%? I'm inclined to say, not only are you dead to me, that person should lose it all. Seriously. You put others accounts in jeopardy as likely what happened with TW.

Haha, yes I agree with this one I would be happy for them to lose 100% of their security deposit and any trade amount. I think in discussion others felt it was harsh.

> **Buyer uses unagreed potentially concerning payment reference that such as 'payment for services', 'house rewire', 'consultancy'** = Haha, yes. Hence need to improve comm's/notifications for traders (not just agents).
> **Buyer uses unagreed innocuous payment reference that such as holiday fund, flowers without agreement.** = this is 5%, seen as less worse than the above for 10%? Because looks like business orientated above vs. personal?

Yes, the first one to me implies the other party is attempting to claim their BTC purchase as an expense so they can off set it against tax. Whatever they want to do is up to them but they should not implicate their counterparty. Sellers do not want to have to explain to their bank why they are receiving payment for invoices etc in a personal or business account. The trade is not a payment for a service it is for BTC. Second part is less of an issue and is fine if communicated and agreed prior to sending payment. 

> **Buyer attempts to debit sellers account using their bank details**
> **Buyer raises request for refund from Revolt / TransferWise due to buyer using their account to sell them Bitcoin.** = buyer losing 100% trade, but keeps security deposit. I'd also suggest losing security deposit.

The above are what the mediators would propose. There needs to be some funds remaining for the buyer to accept mediation. If seller is not happy with buyer not losing everything they can take it to arbitration and make a case for this.

> **Buyer does not make payment in trade window** = don't want to incentivise escalations. Would preface if no comms from willing to wait. e.g. TW transfer and went 4 days instead of 1, through no fault of my own. Other was ok to wait, but if this existed, might have wanted to raise to get %.

Payment method should be set a maximum time amount. TW has been updated to 4 days. I think most of the reasons for late payment can be attributed to buyer not having funds available or buyer not having payment account set up correctly. Eg they see a TW offer they like, register for TW but fail to pass KYC in time to make payment. Meanwhile they have set up an account on Bisq and accepted the offer. Not having funds, and not having an account able to make payment should both be penalized.

**Comment by RififiCastorjunior** [comment]
So if i'm not wrong it will be 25% of 0.001 BTC for trade penalties, woow that not munch for a penalty.

Like someone write in bisq.community we should keep thing simple. Like it's traders who chat and come to an agreement and then if things escalate mediator and arbitrator take turn.

**Comment by pazza83** [comment]
Hi @RififiCastorjunior yes I agree traders should always try and resolve in chat first.

Mediator is needed when things cannot be resolved with chat alone and a different payout amount is needed.

Arbitrator should be last resort. I think most trades in my experience that end there are when other trader is unresponsive. 

>So if i'm not wrong it will be 25% of 0.001 BTC for trade penalties, woow that not munch for a penalty.

It would vary between 5% - 125% of trade amount. Depending on what the issue was. Penalties would likely reduce for small trades as a result of minimum security deposit being decreased.

**Reply by MwithM** [reply]
The problem is that buyers know they'll get 25% of trade amount as standard penalty from sellers, buyers won't accept any worse deal with sellers.
That's why I think that a range of penalties for different categories of violations (minor rule, severe and fraud attempts) it's better. In example, minor violations could go from 2 to 10% of trade amount, severe from 5 to 30% and fraud attempts should make the violator to lose everything.

**Reply by Conza88** [reply]
Is the comm's advising to try resolve via trader chat first, good enough? 
Same with mediator?

**Reply by pazza83** [reply]
> The problem is that buyers know they'll get 25% of trade amount as standard penalty from sellers, buyers won't accept any worse deal with sellers.

25% would only be in cases where seller has not sent BTC, or has sent BTC 3 days late. In those cases I think buyer should expect 25% of trade amount as a standard penalty.

Problem with 2% trade penalties is the volatility of BTC price. If as above seller is 3 days late releasing funds BTC price could have moved by more than 2%. I think penalties should not be associated to BTC price movement but instead fixed to late payment / release or BTC funds.

**Reply by pazza83** [reply]
> Is the comm's advising to try resolve via trader chat first, good enough?

A lot of time things can not be resolved in trader chat.

From my experience these are typical conversations:

Seller > Hi the trade window has closed and I have not received payment
Buyer >  Sorry I am having problems with my bank
Seller >  Ok thanks for letting me know I will open mediation. There will be a penalty for late payment.

---

Buyer >  Hi sorry I cannot make payment with my account. 
Seller >  No problems, there are still 3 days left till the trade window closes do you have another account you can make payment from?
Buyer >  No sorry. I am not going to be able to make payment.
Seller > Thanks for letting me know I will open mediation. There will be a penalty for not completing the trade.

---

Seller > Hi the trade window has closed and I have not received payment
Buyer > [no response]

---

It would be great if things could be resolved in trader chat but I think if they cannot make payment in time, or cannot make payment at all, the trade will require mediation.

95% of trades go through perfectly no need for mediation. Of the 5% that don't almost all require mediation.

**Reply by MwithM** [reply]
> 25% would only be in cases where seller has not sent BTC, or has sent BTC 3 days late. In those cases I think buyer should expect 25% of trade amount as a standard penalty.

Maybe traders don't mind very much waiting for 3 days, but if they know for sure they're gonna get a fixed 25% of trade amount, they won't agree for any other amount with their peer.

> Problem with 2% trade penalties is the volatility of BTC price. If as above seller is 3 days late releasing funds BTC price could have moved by more than 2%. I think penalties should not be associated to BTC price movement but instead fixed to late payment / release or BTC funds.

Yes, maybe 2% is a very small amount considering BTC volatility and that traders can delay accepting the mediation, but that's another reason why using a range of penalties for the table is reasonable.

**Reply by pazza83** [reply]
> Maybe traders don't mind very much waiting for 3 days

Yes I think this is the downside to having guidelines, there is less of a reason for traders to just accept the problems that occur as part of trading. However my opinion is the benefits of having guidelines outweighs going into a trade not knowing what to expect if it does not go to plan. 

With regards a range of penalties I think 5-15% are suitable for fiat trades, maybe less so with altcoins where the volatility would not be so extreme compared with BTC.

**Reply by RififiCastorjunior** [reply]
> Seller > Hi the trade window has closed and I have not received payment
> Buyer > [no response]
> 
> It would be great if things could be resolved in trader chat but I think if they cannot make payment in time, or cannot make payment at all, the trade will require mediation.

I still think that all penalties should be up to the traders, even in mediation. If i'm in a rush for some payment in fiat and just want to trade on Bisq and things take 3 days after the terms of the trade. That could impact all conversations and mediation for this trade. And if the other pair doesn't complete the term of the trade in time and take more time to respond. Heavy penalties should incurs.



@pazza83 is it possible if a trade go to mediation to ask for a heavy penalty with this chart ?

**Reply by MwithM** [reply]
> > Maybe traders don't mind very much waiting for 3 days
> 
> Yes I think this is the downside to having guidelines, there is less of a reason for traders to just accept the problems that occur as part of trading. However my opinion is the benefits of having guidelines outweighs going into a trade not knowing what to expect if it does not go to plan.
> 
> With regards a range of penalties I think 5-15% are suitable for fiat trades, maybe less so with altcoins where the volatility would not be so extreme compared with BTC.

I agree with that too, and that's why I suggest to suggest ranges of penalties and, specially at the beginning, don't be afraid on making them very wide and have big warnings that these are only for guidance reasons and not a strict policy.

**Reply by pazza83** [reply]
> @pazza83 is it possible if a trade go to mediation to ask for a heavy penalty with this chart ?

Yes, but it would only be proposed if their was fault at the buyers or sellers end.

In your example you are in a rush for in a rush for some payment in fiat and just want to trade on Bisq. You choose a 24 hour payment method. If buyer send funds 3 days late you would be awarded 15% of their security deposit.

>That could impact all conversations and mediation for this trade. 

It should impact it. You would say to the late buyer sorry due to the late payment their will be a 15% penalty. Mediator would propose this. Buyer can either accept or reject proposal. If they accept the trade would go to arbitration.

**Comment by pazza83** [comment]
Hi @huey735 please can you post the codified table of trade penalties you created?

**Reply by huey735** [reply]
@pazza83 https://github.com/bisq-network/support/blob/master/docs/trade-penalties.md
